apps/server/src/apps/tldraw-console.app.ts

```diff
@@ -0,0 +1,31 @@
+/* istanbul ignore file */
+import { BootstrapConsole } from 'nestjs-console';
+import { TldrawConsoleModule } from '@src/modules/tldraw';
+
+async function run() {
+	const bootstrap = new BootstrapConsole({
+		module: TldrawConsoleModule,
+		useDecorators: true,
+	});
+
+	const app = await bootstrap.init();
+
+	try {
+		await app.init();
+
+		// Execute console application with provided arguments.
+		await bootstrap.boot();
+	} catch (err) {
+		// eslint-disable-next-line no-console, @typescript-eslint/no-unsafe-call
+		console.error(err);
+
+		// Set the exit code to 1 to indicate a console app failure.
+		process.exitCode = 1;
```

Comment:
Why not directly go out over process.exit(1);

Reply:
fixed